game.py

```diff
@@ -0,0 +1,38 @@
+"""MAIN LOGIC FOR GAME"""
+from models import *
+
+
+def get_enemy_instance(level=1):
+    return Enemy(level)
+
+
+def play():
+
+    player = get_player_instance()
+    enemy = get_enemy_instance()
+
+    while True:
+        try:
+            player.attack(enemy)
+            player.defence(enemy)
+
+        except EnemyDown:
+            player.score += 5
+            enemy = get_enemy_instance(enemy.level + 1)
+
+        except GameOver:
+            def get_scores(player_obj: Player) -> None:
```

Comment:
And... that if just delete the inner function and handle score-write without it?